examples/knowledge_tuning/data-generation-with-llama-70b/data-generation-with-llama-70b.py

```
@@ -0,0 +1,236 @@
+
+with open(output_file, "w") as f:
+    # Write the source document first
+    f.write(f"### Document \n{generated_data[0]['document']}")
+    
+    # Compare generated Q&A pairs
+    for i in range(min(len(generated_data), len(generated_data_mistral))):
+        f.write("Example #{}\n".format(i+1))
+        
+        # LLaMA 3.3 results
+        f.write("### Result from llama3.3\n")
+        f.write(generated_data[i]['question'] + "\n")
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)